Flow.Launcher.Infrastructure/Image/ImageLoader.cs

```diff
@@ -113,7 +114,25 @@ private static ImageResult LoadInternal(string path, bool loadFullImage = false)
                 {
                     return new ImageResult(ImageCache[path], ImageType.Cache);
                 }
-
+                if (path.StartsWith("http://") || path.StartsWith("https://"))
+                {
+                    // Download image from url
+                    using (WebClient client = new WebClient())
+                    {
+                        using (MemoryStream stream = new MemoryStream(client.DownloadData(path)))
+                        {
+                            var image = new BitmapImage();
+                            image.BeginInit();
+                            image.CacheOption = BitmapCacheOption.OnLoad;
+                            image.StreamSource = stream;
+                            image.EndInit();
+                            image.Freeze();
```

Comment:
I am curious what if we remove this line?

Reply:
It crashes the entire app XD